wallet/chainntfns.go

```diff
@@ -96,6 +98,16 @@ func (w *Wallet) handleChainNotifications() {
 			var err error
 			switch n := n.(type) {
 			case chain.ClientConnected:
+				// Before attempting to sync with our backend,
+				// we'll make sure that our birthday block has
+				// been set correctly to potentially prevent
+				// missing relevant events.
+				if err := w.birthdaySanityCheck(); err != nil {
+					log.Errorf("Unable to sanity check "+
```

Comment:
We may actually want to panic here. As is, the loop will exit, and we'll silently stop receiving any chain notifications.

Reply:
Fixed.